src/Middleware/SpaServices.Extensions/src/ReactDevelopmentServer/ReactDevelopmentServerMiddleware.cs

```diff
@@ -62,9 +63,10 @@ internal static class ReactDevelopmentServerMiddleware
         }
 
         private static async Task<int> StartCreateReactAppServerAsync(
-            string sourcePath, string scriptName, string pkgManagerCommand, ILogger logger)
+            string sourcePath, string scriptName, string pkgManagerCommand, int portNumber, ILogger logger)
         {
-            var portNumber = TcpPortFinder.FindAvailablePort();
+            if (portNumber == 0)
+                portNumber = TcpPortFinder.FindAvailablePort();
```

Comment:
We generally use curly braces even when there's just one line. It prevents confusion if more statements need to be added later, or if this line is removed and so on.

Reply:
✔️